plugins/woocommerce/changelog/add-shipping-calculator-filters

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Hmm. I thought all levels were included, with the exceptions of a patch that has no changelog text and an explanatory comment field.

Reply:
Just confirming this, here is a recent-ish example of a [patch-level changelog entry](https://github.com/woocommerce/woocommerce/blob/1012c45a4869d32d0f5b398c628180297cc3b92c/plugins/woocommerce/changelog/fix-bacs-ui), and the [corresponding entry](https://github.com/woocommerce/woocommerce/blob/8.2.0/plugins/woocommerce/readme.txt#L280) in the changelog section of the readme.

Reply:
```suggestion
Significance: minor
```
Having written all that out ... actually I will change it to `minor` as you suggested, since `minor` is used when adding (or deprecating) functionality in a backwards-compatible manner whereas `patch` is for backwards-compatible bug fixes (and this is not a bug fix). Which I guess is what you were getting at in the first place 🙃